streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/register/impl/StormServiceRegistrar.java

```diff
@@ -37,69 +39,65 @@
 public class StormServiceRegistrar extends AbstractServiceRegistrar {
     public static final String COMPONENT_STORM_UI_SERVER = ComponentPropertyPattern.STORM_UI_SERVER.name();
     public static final String COMPONENT_NIMBUS = ComponentPropertyPattern.NIMBUS.name();
-    public static final String PARAM_STORM_UI_SERVER_HOSTNAME = "uiServerHostname";
-    public static final String PARAM_NIMBUS_HOSTNAMES = "nimbusesHostnames";
-    public static final String PARAM_NIMBUS_PRINCIPAL_NAME = "nimbusPrincipalName";
-    public static final String STORM_ENV = ServiceConfigurations.STORM.getConfNames()[1];
+
+    public static final String PARAM_NIMBUS_SEEDS = "nimbus.seeds";
```

Comment:
That represents config key in UI. If you think config key is better than param, we can change it.